### DISCUSSION TITLE
Bulk Adding and Removing Tags

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#2497] [Ideas] [answered]
It would be nice if Monkeytype had a way to bulk add or remove tags based on certain criteria or if you clicked on multiple at once. Say for example, if I wanted to give all tests I completed in 2021 a `2021` tag or replace all the tests with the tag `x` with the tag `y`. It would be hard to do it manually because I've completed more than 4000 tests in total. Searching and or Filtering by tag would also be great!

**Comment by zachpoblete** [comment]
Although MT now only displays the last 1000 completed tests, this would still be a feature I would really like to see implemented.

**Comment by unrealapex** [comment, marked as answer]
An issue has been filed for this: https://github.com/monkeytypegame/monkeytype/issues/3324.
